volunteer.html

```diff
@@ -70,8 +70,8 @@ <h3 class="text-center">Development</h3>
 <div class="panel">
   <div class="panel-container">
     <div class="panel-fill text-center row">
-      <div class="column l-6-12 l-offset-3-12">
         <a name="signup"></a><h2>Want to volunteer?</h2>
+      <div class="column m-8-12 m-offset-2-12 text-left">
```

Comment:
Please can we fix the indentation of the previous line.

Reply:
fixed in 5a1219a220d4e74aa963c6b5d7bd4c34643cc123